h/groups/views.py

```diff
@@ -159,9 +159,31 @@ def join(request):
     url = request.route_url('group_read', hashid=group.hashid, slug=group.slug)
     return exc.HTTPSeeOther(url)
 
+@view_config(route_name='group_leave',
```

Comment:
Nitpicky McNitPick here: two blank lines before a function definition in Python land.